solutions/account-infrastructure-base/main.tf

```diff
@@ -2,6 +2,167 @@
 # Base Account
 ########################################################################################################################
 
+locals {
+  target_service_details = {
+    "IAM" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "apprapp" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "cloud-object-storage" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "codeengine" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "compliance" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "container-registry" = {
+      enforcement_mode = var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "containers-kubernetes" = {
+      enforcement_mode = var.cbr_enforcement_mode == "report" ? "disabled" : var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "containers-kubernetes-cluster" = {
+      enforcement_mode = var.cbr_enforcement_mode == "report" ? "disabled" : var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "containers-kubernetes-management" = {
+      enforcement_mode = var.cbr_enforcement_mode == "report" ? "disabled" : var.cbr_enforcement_mode
+      global_deny      = false
+    }
+    "context-based-restrictions" = {
```

Comment:
Setting enforcement to `enabled`  for CBR service causing an error to prevent lockout
```
│         "errors": [
│             {
│                 "code": "rule_self_lock_out_error",
│                 "message": "Request forbidden to avoid self lock-out via rule 'b7fd52d0a324fea5a7ba6feeb6017397' from client IP address '101.0.63.188'"
│             }
│         ],
```

may need to remove this service as well. 
cc: @vburckhardt